.prettierrc.yaml

```diff
@@ -1 +1,2 @@
 printWidth: 85
+endOfLine: auto
```

Comment:
How about trying `.gitattributes` to get rid of CRLF for good https://prettier.io/docs/en/options.html#end-of-line (item no 4.)?
> Add `* text=auto eol=lf` to the repo's .gitattributes file. You may need to ask Windows users to re-clone your repo after this change to ensure git has not converted LF to CRLF on checkout.

Reply:
Sure, but that would not prevent prettier on Windows to change it back as soon as you run it? Or does prettier read and respect the .gitattributes files as well @VitGottwald ?

Reply:
The default value for endOfLine is `nl`. If we keep the default prettier should use `nl` when we run it. 

There might be a problem with git though. Old branches that do not have `.gitattributes` yet - when checked out may introduce CRLF back. I believe the process to get rid of CLRF would be to add `.gitattributes` to master, then merge master to all outstanding branches and then do a new clone of the repository.

[Here] is a doc page from github about how to do this, maybe it us not necessary to clone again. Would you like to try it @phaumer I should I do it?

Reply:
I wonder why `npm run prettier` would replace all LFs with CRLFs`. Will have to get access to a windows box to try it out.

Reply:
I just tried this on a windows machine:

* By default files are checked out with CRLF
* When `endOfLine` is set to default `lf`, running  `npm run pretty` converts the line endings from CRLF to `lf` (as expedted)
* One can validate the above by opening a file in VSCode and checking the current line endings in the _status bar_ at the bottom. For files touched by prettier (e.g. package.json) it will be `LF` and for files not touched by prettier (e.g. LICENSE.txt) it will remain as `CRLF`.
* When `git diff` is run, it prints a warning message that _LF will be replaced by CRLF in file.xyz_ for all files that prettier touched. This is somewhat confusing message from git because if these files were (deleted from the working directory and) checked out now, they would get a CRLF instead of the current LF.
* Adding `.gitattributes` with `* text=auto eol=lf` will make git happy and it will not issue the warning messages (beause now if a file is deleted and checked out, it will get LF, like it has after the prettier run).